src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -1480,6 +1480,17 @@ class ActorIdStringGenerator : public Generator<std::string> {
     std::string _actorId;
 };
 
+/** `{^NowTimestamp: {}}` */
```

Comment:
I have a feeling this could be more general. Let me know if I'm wrong with my thinking here, but isn't this closer in spirit to a type conversion of an existing set of generators we have? Aren't we going to need to add `^RandomTimestamp`, `^Timestamp` and `^IncTimestamp`? I think a decent API is something along the lines of `{^ConvertToTimestamp: {from: ^Now: {}}}` to copy the integer cast cases.

Reply:
This change is to test the performance of Atlas stream processing. Just to give the background, the `Passthrough_ChangeStreamSource.yml` does processing on the documents that are inserted into a collection. The idea behind adding the `timestamp` is to keep track of the point `(StartTimestamp)` in time when we started the insertion. The stream processing job will start consuming the documents for processing from `(StartTimestamp)`, after all the documents are ingested.

This allows us to track the performance of the stream processing job without the delay caused by inserting documents in the collection.

The `timestamp` doesn't need to be unique.